### DISCUSSION TITLE
CasaOS: una buona soluzione?

### DISCUSSION THREAD

**Discussion by angio89** [indomus/forum#3100] [Nuove idee, proposte, progetti e guide]
Buongiorno a tutti,
mi ha stupito, dopo una ricerca qui sul forum, che nessuno abbia mai parlato di CasaOS, il sistema operativo che sembra realizzato apposta per la domotica e Home Assistant... o così sembra!
Infatti, la mia domanda è proprio questa: qualcuno lo ha mai provato? Se sì, ha dei riscontri positivi e/o negativi da condividere?

Sto per acquistare un miniPC come hub personale per la mia domotica di casa ed ho letto fra le discussioni di Telegram un utente che chiedeva informazioni riguardo questo sistema operativo ma senza una risposta. Provo a scrivere quindi qui sul forum per capire se qualcuno può aiutarmi.

Grazie alla demo online è possibile provare il sistema operativo ed "a pelle" sembra fantastico ma vorrei avere delle opinioni da parte di qualcuno più esperto.

Grazie a tutti per le eventuali risposte!

**Comment by angio89** [comment]
Viste le scarse risposte al mio thread ho deciso di provare la via di questo sistema operativo e di fare un po' da cavia. Ho scoperto nel frattempo che esistono anche altre soluzioni simili a CasaOS, con lati positivi e negativi rispetto a quest'ultimo.

Non appena avrò informazioni sufficienti vi aggiornerò sperando possa servire a qualcuno.

**Comment by federoco** [comment]
Ciao Angio89, a me interesserebbe approfondire l'argomento.
Grazie
